.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+name: Bedrock Node.js CI
+
+on: [push]
+
+jobs:
+  test-node:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+        mongodb-version: ['4.2']
```

Comment:
not needed.  I will add... that I don't know where the matrix mongodb pattern came from, but I don't think we have a use case for that at this time.